tests/rptest/services/redpanda.py

```diff
@@ -1201,6 +1201,12 @@ def set_cluster_config(self, values: dict, expect_restart: bool = False):
             raise AssertionError(
                 "Nodes report restart required but expect_restart is False")
 
+    def get_cluster_config(self,
+                           node: Optional[ClusterNode] = None,
+                           include_defaults: Optional[bool] = None):
+        return self._admin.get_cluster_config(
+            node, include_defaults=include_defaults)
```

Comment:
In order to not load down the RedpandaService object with arbitrary helpers, it would probably be better to create an `admin = Admin(self.redpanda)` object in your test class. This is a good case, too: it isn't really using anything directly from the RedpandaServer, just the admin instance that exists.